GodotDebugSession/GodotDebuggerSession.cs

```diff
@@ -88,10 +88,11 @@ protected override async void OnRun(DebuggerStartInfo startInfo)
                         int remoteDebugPort = ((IPEndPoint)remoteDebugListener.LocalEndpoint).Port;
 
                         // Launch Godot to run the game and connect to our remote debugger
+                        string extraArgs = string.Join(" ", godotStartInfo.ExecutableArguments);
```

Comment:
Here each argument needs to be quoted if it contains spaces.

Reply:
Would it be better to only quote arguments with spaces or quote every argument?

Reply:
I don't know if quoting arguments without spaces may cause problems with some applications, but since this is only for Godot it should be fine.

Reply:
I wasn't satisfied with the way I was escaping the arguments which would not handle some edge cases (like having quotes in the arguments) and I thought it may be better to use an external library (like **MedallionShell**) to parse the arguments, so I opened #21.

Feel free to use whichever PR you prefer or let me know if there is another way in which you want me to fix this instead.